external/vcm/vcm/derived_mapping.py

```diff
@@ -37,19 +43,32 @@ def register(cls, name: Hashable, required_inputs: Iterable[Hashable] = None):
                 Only the direct dependencies need to be listed here. e.g. if derived
                 variable "a" requires "b", and "b" requires "c", the required_inputs
                 for "a" should just be ["b"].
+            use_nonderived_if_exists:
```

Comment:
The `required_inputs` arg description part about the intended usage of the registry still seems somewhat contradictory to what this variable does.

Reply:
I removed the confusing part of the `required_inputs` docstring:
> "Even if the requirements are not well-defined, they should
                still be listed. (e.g. dQu only needs dQxwind, dQywind if dQu is not
                in the data). This is because the usage of this registry is for when
                an output is explicitly requested as a derived output variable and thus
                it is assumed the variable does not already exist and needs to
                be derived.

since I think removing the try/except from the derived var functions makes it a bit clearer that these are for the case when the variable is derived.